guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
-
-### Lightning address
-
-[Lightning addresses](https://lightningaddress.com/), which look like regular emails (jane@domain.com), are reusable ways users can receive Lightning payments. These are not native to the Lightning network itself, but rather offered by a trusted server often ran by the wallet provider.
-
-Generating a Lightning address should be simple for users and done within a profile page or during the users [first use]({{ "/guide/daily-spending-wallet/first-use/" | relative_url }}). At a minimum, your wallet should support sending to Lightning addresses, so it's interoperable with other wallets.
+Any fees the user may incur should be communicated alongside payment requests so users will know in advance if they will be charged before receiving a payment. What these fees are for will vary from app to app but they a generally for opening channels. Below are common fee scenarios this wallet will face:
 
+- A fee is charged if the user has no open channels and needs one opened [on-demand]({{ "/guide/how-it-works/lightning-services/#on-demand-liquidity" | relative_url }}) when the payment is received.
+- A fee may be charged when requesting a zero-amount invoice if the received amount is higher than their [inbound liquidity]({{ "/guide/how-it-works/liquidity/" | relative_url }}), also known as their receive limit.
+- A fee is charged if the user is requesting an amount higher than their receive limit.
+- No fee is charged if the user is requesting an amount lower than their receive limit.
+ 
 <div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/my-profile-empty.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/my-profile-empty@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
    layout = "shadow"
-   caption = "A users profile is a great place to generate a new Lightning address."
-   alt-text = "Screen showing users profile."
+   caption = "Let users know if fees will be charged."
+   alt-text = "Screen showing a payment request with setup fees shown."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/generate-lightning-address.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/generate-lightning-address@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/SetupFee.png"
```

Comment:
Labels are called `Tags` in the activity page, would be good to keep that consistent. Same with the user of the tag and note icons, so people can recognize them immediately. Not a big deal, we can make that consistent when we do the screen review after the content updates.